context/condarc

```diff
@@ -1,7 +1,6 @@
 auto_update_conda: False
 channels:
   - rapidsai-nightly
-  - dask/label/dev
```

Comment:
@raydouglass Is this supposed to be reverted, too?

Reply:
Yes, good catch!